tests/TestableIO.System.IO.Abstractions.TestingHelpers.Tests/MockFileSystemTests.cs

```diff
@@ -435,6 +435,26 @@ public void MockFileSystem_Constructor_ThrowsForNonRootedCurrentDirectory()
             );
             Assert.That(ae.ParamName, Is.EqualTo("currentDirectory"));
         }
+        
+        [Test]
+        public void MockFileSystem_Constructor_ShouldSupportDifferentRootDrives()
+        {
+            var fileSystem = new MockFileSystem(new Dictionary<string, MockFileData>
+            {
+                [@"c:\"] = new MockDirectoryData(),
+                [@"z:\"] = new MockDirectoryData(),
```

Comment:
What happens, when you add a nested directory in a new drive, e.g. `z:\foo\bar`?
Could you adapt the test, to also verify that the drive is created in this case?

Reply:
added more tests to cover those scenarios